src/Servers/Kestrel/shared/test/HttpParsingData.cs

```diff
@@ -201,11 +196,18 @@ public static IEnumerable<string> RequestLineInvalidData
                     "CUSTOM / HTTP\r\n",
                     "CUSTOM / HTTP/\r\n",
                     "CUSTOM / HTTP/1\r\n",
-                    "CUSTOM / HTTP/1.\r\n",
```

Comment:
We're adding these tests cases back, right? We can assert it's a 505 now. I understand if you only want to change the behavior in `main`, but I don't think it's risky.